contrib/stable/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# stable/Dockerfile
+#
+# Build a Skopeo container image from the latest
+# stable version of Skopeo on the Fedoras Updates System.
+# https://bodhi.fedoraproject.org/updates/?search=skopeo
+# This image can be used to create a secured container
+# that runs safely with privileges within the container.
+#
+FROM registry.fedoraproject.org/fedora:latest
```

Comment:
We should be careful with using the latest tag:
```
~ $ podman run --rm fedora:32 cat /etc/os-release|head -n2
NAME=Fedora
VERSION="32 (Container Image)"
~ $ podman run --rm fedora:latest cat /etc/os-release|head -n2
NAME=Fedora
VERSION="31 (Container Image)"
```
Latest is pointing to an F31 image.

Reply:
```console
% ./skopeo --override-os linux inspect  docker://registry.fedoraproject.org/fedora:latest
…
        "version": "32"
…
        "DISTTAG=f32container",
        "FGC=f32",
```
Is it possible that you have a previously-pulled `fedora:latest` on your system?

Reply:
I'll second that, latest is pulling 32 for me from `registry.fedoraproject.org/fedora`.  I'd really rather not put in a hard version into the file.

Reply:
OTOH, looking at the `Dockerfile`, what are the chances it will continue to work, and be optimal, for any future release without changes?

(I guess it’s better to have a `:latest` base that may break the image, if we notice, than a `:32` base that may silently continue to work for years without being updated. But it’s not a slam dunk.)

Reply:
> Is it possible that you have a previously-pulled fedora:latest on your system?

Nope, that's a fresh `fedora:latest` but I now saw that I was pulling from `docker.io`. The one from the Fedora registry being used here is good :+1: 

I still cringe when seeing `:latest` tags for anything other than testing. I was "raised" with the policy to never use latest in production, see the K8s docs:

> Note: You should avoid using the :latest tag when deploying containers in production as it is harder to track which version of the image is running and more difficult to roll back properly.

If something breaks, it should break for all equally. IMHO, the responsibility of testing should be on our side (maintainers) and not the users side.

Reply:
UNCLE!  Switch to `:32`